docs/managing-files/previewer.rst

```diff
@@ -0,0 +1,95 @@
+
+..
+    This file is part of Invenio.
+    Copyright (C) 2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Previewing files
+================
+Invenio has support for previewing many of the most popular file formats
+including PDF, ZIP, Markdown, images and Jupyter Notebooks. In this section we
+will discuss how previewing files work in Invenio and which options are
+available.
+
+`Invenio-Previewer <https://invenio-previewer.readthedocs.io/en/latest/>`_
+allows you to register previewers based on the file extension. To preview
+files, ensure that a preview endpoint have been configured in your
+``config.py``, like this:
+
+.. code:: python
+
+    RECORDS_UI_ENDPOINTS=dict(
+        recid_previewer=dict(
+            pid_type='recid',
+            route='/records/<pid_value>/preview/<filename>',
+            view_imp='invenio_previewer.views:preview',
+            record_class='invenio_records_files.api:Record',
+        ),
+    )
+
+Upload a file (see :ref:`managing-files`) of a type supported by the previewer,
```

Comment:
Once, there is a full `managing-files` section, we should update the reference to point to the exact file upload step.